### PR TITLE
Optionally enable health check with `HEALTH_CHECK_PATH` env var

### PR DESCRIPTION
**Pull request by epiphone**
Addresses https://github.com/colyseus/proxy/issues/17.

### CODE REVIEW

**Comment by pnutmath** [comment]
Kubernetes have multiple health-check endpoints for different purpose https://kubernetes.io/docs/reference/using-api/health-checks/

File: proxy.ts [hunk 1]
```diff
@@ -116,6 +116,18 @@ getNodeList().
...
 
 const reqHandler = (req: http.IncomingMessage, res: http.ServerResponse) => {
+  const healthCheckPath = process.env.HEALTH_CHECK_PATH;
+  if (
+    healthCheckPath &&
+    (req.url === healthCheckPath ||
+      (healthCheckPath[0] === "*" &&
+        req.url?.endsWith(healthCheckPath.slice(1))))
+  ) {
+    res.statusCode = 200;
```
Comment (pnutmath) [inline proxy.ts:126]: Simply responding 200 status might not satisfies all use cases of health check.

**Comment by epiphone** [comment]
@pnutmath I recommend opening another PR for any extended features.

**Comment by pnutmath** [comment]
@epiphone We can add health check at colyseus server level... which will proxy through colyseus proxy.. No need of separate health check here

**Comment by epiphone** [comment]
I think a separate health check is useful: Proxy and Server processes can run on separate instances and may be independently scaled.

**Comment by endel** [comment]
Thank you for the PR @epiphone 🙏 , just merged https://github.com/colyseus/proxy/pull/35 that contains your changes + other feedback. Cheers!

**Pull request closed without merging**